content/extending-ui/extending-agent-app/agent-app-sdk/index.mdx

```diff
@@ -176,14 +176,15 @@ widget.on("customer_profile", profile => {
-| `id`          | Unique ID of a visitor                                                                                                                               |
-| `name`        | Visitor name (if provided)                                                                                                                           |
-| `email`       | Visitor email (if provided)                                                                                                                          |
-| `chat`        | Object with two properties: `id` (unique chat id) and `groupID` (unique group id); this object may be empty when a visitor is not currently chatting |
-| `source`      | String representing the source of an event; possible values: `chats`, `customers`, `archives`                                                        |
-| `geolocation` | Object containing detailed information about customer's geolocation                                                                                  |
+| Property          | Description                                                                                                                                                                                 |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`              | Unique ID of a visitor                                                                                                                                                                      |
+| `name`            | Visitor name (if provided)                                                                                                                                                                  |
+| `email`           | Visitor email (if provided)                                                                                                                                                                 |
+| `chat`            | Object with two properties: `id` (unique chat id) and `groupID` (unique group id); this object may be empty when a visitor is not currently chatting                                        |
+| `source`          | String representing the source of an event; possible values: `chats`, `customers`, `archives`                                                                                               |
+| `geolocation`     | Object containing detailed information about customer's geolocation                                                                                                                         |
+| `customVariables` | Object containing [custom variables](https://www.livechatinc.com/help/custom-variables-configuration/) for given customer/chat; this object may be empty when custom variables were not set |
```

Comment:
```suggestion
| `customVariables` | Object containing [custom variables](https://www.livechatinc.com/help/custom-variables-configuration/) for a given customer/chat. This object may be empty when custom variables were not set |
```

Reply:
`This object may be empty when custom variables were not set` - is this sentence necessary? Maybe we could add `(If provided)`? If you really want to keep this sentence, I think it should be:
`This object is empty when custom variables weren't set.` - because if they weren't set, the object is **always** empty, not **may be empty**

Reply:
@ale-kacperczyk changed in 92966a9